usng4j-api/src/main/java/org/codice/usng4j/CoordinateSystemTranslator.java

```diff
@@ -167,4 +167,10 @@ UsngCoordinate toUsng(
    * @throws ParseException when 'msgrsString' isn't in MGRS format.
    */
   UsngCoordinate parseMgrsString(String mgrsString) throws ParseException;
+
+  /**
+   * @param decimalDegreesCoordinate longitude latitude coordinate
+   * @return true if latitude is in UPS space
+   */
+  boolean isInUPSSpace(DecimalDegreesCoordinate decimalDegreesCoordinate);
```

Comment:
❓ Do we already have a method stub for this? 
https://github.com/codice/usng4j/blob/master/usng4j-api/src/main/java/org/codice/usng4j/UtmUpsCoordinate.java#L40

Reply:
That's in UtmUpsCoordinate which is only available after a unit conversion. The lat/lon coordinate is what should be used to determine that which is available before conversion.